loleaflet/src/control/Control.ContextMenu.js

```diff
@@ -169,8 +169,8 @@ L.Control.ContextMenu = L.Control.extend({
 	},
 
 	_amendContextMenuData: function(obj) {
-		// Add a 'delete' entry for mobile for graphic selection.
-		if (this._map._docLayer.hasGraphicSelection() && window.mode.isMobile()) {
+		// Add a 'delete' entry  for graphic selection when on mobile browser or in a mobile app.
+		if (this._map._docLayer.hasGraphicSelection() && (window.ThisIsAMobileApp || window.mode.isMobile())) {
```

Comment:
Tor, is there a particular reason why this check has to be done against whether it's a mobile app, and not against whether it's a tablet? I presume this fix won't work in web-based online on tablets.